test/reporter-stdout.js

```diff
@@ -0,0 +1,21 @@
+/*globals QUnit:true*/
+/*jshint node:true*/
+var QUnit = require( "../dist/qunit" );
+
+// Options: { output: "minimal" || "verbose" }
+QUnit.stdout();
+
+// Load QUnit tests
+require( "./logs" );
```

Comment:
It will use the QUnit variable assigned above. This current file `test/reporter-stdout.js` will work as a context module for its following requirements.